### DISCUSSION TITLE
How to generate `collection/index.html`?

### DISCUSSION THREAD

**Discussion by firedev** [bridgetownrb/bridgetown#809] [Q&A] [answered]
Take a look at Bridgetown documentation and its `/docs` collection. 

* https://www.bridgetownrb.com/docs/ ← ✅ This works
* https://www.bridgetownrb.com/docs.html ← ✅ Because this file exists
* https://www.bridgetownrb.com/docs/index.html ← 🛑 Doesn't work despite `src/_docs/index.md`

My server doesn't serve `/docs.html` as `/docs`, is it possible to generate `/collection/index.html` instead of `/collection.html`? `permalink` is set as follows:

```
    permalink: "/:collection/:path.*"
```

**Comment by firedev** [comment]
Okay I figured it's the way it is, had to solve it on the server-side, `try_files` in `Caddyfile` did it:
```
	file_server
	try_files {path} {path}.html
```

Unfortunately after deploy it didn't work.

**Comment by jaredcwhite** [comment, marked as answer]
Try changing the `:path.*` to `:path/` in your permalink settings.

**Reply by firedev** [reply]
Thank you! Also I think this `Caddyfile` config works with all variants:

```
{   ...
	file_server
	try_files {path} {path}.html {path}/index.html
	try_files {path}/ {path}.html {path}/index.html
}
```
